lib/es/client.rb

```diff
@@ -16,16 +23,22 @@ def initialize(service)
       private
 
       def build_client
-        if dev_env?
-          Elasticsearch::Client.new(config)
-        else
-          Elasticsearch::Client.new(config) do |f|
+        cache_expires_in = ENV.fetch('CACHE_EXPIRY', 5).to_i
```

Comment:
I would really like to avoid hitting the elastic search cluster where possible but it's nice to have fresh data updates in the UI as well. 

Thoughts appreciated about 5 mins being too long here?

Reply:
I'm pretty sure five minutes is fine. I don't think there are any use cases where we'd need more frequent updates than that.

Reply:
thanks - if we do we can inject a env value to modify it.